src/Reporter.cpp

```diff
@@ -417,6 +417,17 @@ namespace geopm
             {"time-hint-unset (s)", {"TIME_HINT_UNSET"}, sample_only},
         };
 
+        auto all_names = m_platform_io.signal_names();
+        if (all_names.find("ENERGY_ACCELERATOR") != all_names.end()) {
```

Comment:
```suggestion
        if (all_names.count("ENERGY_ACCELERATOR") == 1) {
```

Reply:
This is slightly more readable?  Not really a required change.  I using find() is more consistent with other use cases in our source code.

Reply:
May not be required, but I think it's cleaner overall.